python_frank_energie/models.py

```diff
@@ -606,8 +605,14 @@ def old_format_delivery_site_as_dict(self):
         sites_as_dict = []
         for site in self.deliverySites:
             address = site.get('address', {})
+            house_addition = (
+                address.get('houseNumberAddition', '')

```

Comment:
If there is no houseNumberAddition it returns '' (after the comma)
So the next line is not necessary
if address.get('houseNumberAddition') else ''